lib/collection.js

```diff
@@ -66,6 +63,51 @@ const updateDocuments = require('./operations/collection_ops').updateDocuments;
+const deleteManySchema = require('./schemas/collection_schemas').deleteManySchema;
+const deleteOneSchema = require('./schemas/collection_schemas').deleteOneSchema;
+const distinctSchema = require('./schemas/collection_schemas').distinctSchema;
+const dropSchema = require('./schemas/collection_schemas').dropSchema;
+const dropIndexSchema = require('./schemas/collection_schemas').dropIndexSchema;
+const dropIndexesSchema = require('./schemas/collection_schemas').dropIndexesSchema;
+const estimatedDocumentCountSchema = require('./schemas/collection_schemas')
+  .estimatedDocumentCountSchema;
+const findSchema = require('./schemas/collection_schemas').findSchema;
+const findAndModifySchema = require('./schemas/collection_schemas').findAndModifySchema;
+const findAndRemoveSchema = require('./schemas/collection_schemas').findAndRemoveSchema;
+const findOneSchema = require('./schemas/collection_schemas').findOneSchema;
+const findOneAndDeleteSchema = require('./schemas/collection_schemas').findOneAndDeleteSchema;
+const findOneAndReplaceSchema = require('./schemas/collection_schemas').findOneAndReplaceSchema;
+const findOneAndUpdateSchema = require('./schemas/collection_schemas').findOneAndUpdateSchema;
+const geoHaystackSearchSchema = require('./schemas/collection_schemas').geoHaystackSearchSchema;
+const indexesSchema = require('./schemas/collection_schemas').indexesSchema;
+const indexExistsSchema = require('./schemas/collection_schemas').indexExistsSchema;
+const indexInformationSchema = require('./schemas/collection_schemas').indexInformationSchema;
+const initializeOrderedBulkOpSchema = require('./schemas/collection_schemas')
+  .initializeOrderedBulkOpSchema;
+const initializeUnorderedBulkOpSchema = require('./schemas/collection_schemas')
+  .initializeUnorderedBulkOpSchema;
+const insertManySchema = require('./schemas/collection_schemas').insertManySchema;
+const insertOneSchema = require('./schemas/collection_schemas').insertOneSchema;
+const isCappedSchema = require('./schemas/collection_schemas').isCappedSchema;
+const listIndexesSchema = require('./schemas/collection_schemas').listIndexesSchema;
+const mapReduceSchema = require('./schemas/collection_schemas').mapReduceSchema;
+const optionsSchema = require('./schemas/collection_schemas').optionsSchema;
+const parallelCollectionScanSchema = require('./schemas/collection_schemas')
+  .parallelCollectionScanSchema;
+const reIndexSchema = require('./schemas/collection_schemas').reIndexSchema;
+const renameSchema = require('./schemas/collection_schemas').renameSchema;
+const replaceOneSchema = require('./schemas/collection_schemas').replaceOneSchema;
+const statsSchema = require('./schemas/collection_schemas').statsSchema;
+const updateManySchema = require('./schemas/collection_schemas').updateManySchema;
+const updateOneSchema = require('./schemas/collection_schemas').updateOneSchema;
+const watchSchema = require('./schemas/collection_schemas').watchSchema;
```

Comment:
just want to register that I think we're getting out of hand with the number of imports in these files 😄, but I think this will all be resolved with the command refactor so I'll hold my tongue for now

Reply:
I definitely agree